docs/deploy-hyperlane.mdx

```diff
@@ -1,113 +1,334 @@
-import TerminologyPartial from "/docs/partials/deploy-hyperlane/_terminology.mdx";
-import SetupKeysPartial from "/docs/partials/deploy-hyperlane/_setup-keys.mdx";
-import DeployContractsPartial from "/docs/partials/deploy-hyperlane/_deploy-contracts.mdx";
-import SendTestMessagesPartial from "/docs/partials/deploy-hyperlane/_send-test-messages.mdx";
+# Deploy Hyperlane to a new chain
 
-# Get Started
-
-Hyperlane can be deployed by anyone to any chain.
-This guide covers sending your first interchain message from a new EVM chain. If you'd like to deploy to a Cosmos appchain, follow this [community-submitted guide](https://github.com/many-things/cw-hyperlane/blob/25a31818e9bf909875ab091f3884ee71bd25814f/DEPLOYMENT.md).
+:::info
+This guide will help you deploy Hyperlane to your new chain as quickly as possible. This includes the core mailbox and ISM contracts as well as warp route contracts for assets you’re bridging
 
-By the end of this guide you will have deployed and configured your Mailbox smart contract and off-chain agents, allowing developers to send interchain messages to and from your chain.
+The guide is intended for testing - by the end you’ll be able to pass messages between your custom chain and any other in the Hyperlane network.
 
-:::note
-
-This guide has been specifically made to demonstrate how to connect a chain that _already has Hyperlane on it_ to a new chain that does _not_ have Hyperlane.
+To go to production, ensure you’ve successfully pushed a registry PR (the last step) and reach back over telegram and we can add your custom chain to the core chains we have in the CLI and relay for.
 
 :::
 
-<TerminologyPartial />
+## Pre-Requisites: Chain Metadata & Funded Wallet
 
-## Overview
+Anyone can begin this quickstart guide once they have the following artifacts & assets available:
+- A new, custom, or pre-existing network of your choice, including the following metadata:
+    - A chain name, e.g. `ethereum`
+    - A chain ID, e.g. `1`
+    - A RPC URL, e.g. `https://eth.llamarpc.com`
```

Comment:
Rather than trying to summarize ChainMetadata here (which may change over time), consider just describing the _concept_ of chain metadata and/or include a link to a Reference page about it.